### PR TITLE
I800 sort collections by title

### PR DESCRIPTION
**Pull request by ShanaLMoore**
# Story

By Default, the sort by dropdown is title A-Z on the Collection Show Page instead of relevance

Refs #800 

# Expected Behavior Before Changes

Screenshot 2023-09-18 at 5 00 12 PM

# Expected Behavior After Changes

Screenshot 2023-09-18 at 16-57-49 Collection __ Hyku Commons

### CODE REVIEW

File: app/controllers/concerns/hyrax/collections_controller_behavior_decorator.rb [hunk 1]
```diff
@@ -10,6 +10,13 @@ def load_member_subcollections
 
       @subcollection_docs.sort_by! { |doc| doc.title.first&.downcase }
     end
+
+    def show
+      # OVERRIDE Hyrax 3.6.0 to default sort by title instead of relevance
+      params[:sort] ||= "title_ssi asc"
```
Comment (kirkkwang) [inline app/controllers/concerns/hyrax/collections_controller_behavior_decorator.rb:16]: what do you think about something like this?

```suggestion
      params[:sort] ||= "#{Hyrax::CollectionSearchBuilderDecorator::SORT_FIELD} asc"
```

File: app/search_builders/hyrax/collection_search_builder_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#frozen_string_literal: true
+
+module Hyrax
+  module CollectionSearchBuilderDecorator
+    # OVERRIDE Hyrax 3.6.0 to use title_ssi instead of title_si (hyrax's default value)
+    def sort_field
+      'title_ssi'
```
Comment (kirkkwang) [inline app/search_builders/hyrax/collection_search_builder_decorator.rb:7]: and this
```suggestion
      SORT_FIELD
    end

    SORT_FIELD = 'title_ssi'.freeze
```

File: app/search_builders/hyrax/collection_search_builder_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+module Hyrax
+  module CollectionSearchBuilderDecorator
+    # OVERRIDE Hyrax 3.6.0 to use title_ssi instead of title_si (hyrax's default value)
+    def sort_field
+      DEFAULT_SORT_FIELD
+    end
+
+    DEFAULT_SORT_FIELD = 'title_ssi'.freeze
+    end
```
Comment (kirkkwang) [inline app/search_builders/hyrax/collection_search_builder_decorator.rb:11]:
```suggestion
```

**Review by alishaevn — APPROVED** [review]

**Comment by laritakr** [comment]
**When the URL does not include a sort parameter, it defaults to title (A to Z)**
Screenshot 2023-09-18 at 10 01 41 PM

**When a sort value is selected from the dropdown, the works are sorted in the selected sequence**
Screenshot 2023-09-18 at 10 01 56 PM

**Review by kirkkwang — APPROVED** [review]
very nice!

**Pull request merged**